web/src/modules/topic/components/Node/EditableNode.styles.tsx

```diff
@@ -1,8 +1,8 @@
 import styled from "@emotion/styled";
 import { TextareaAutosize } from "@mui/material";
 
-export const nodeWidth = 300;
-export const nodeHeight = 84;
+export const nodeWidth = 256;
+export const nodeHeight = 66;
```

Comment:
what do you think about going even smaller? https://github.com/amelioro/ameliorate/pull/115#discussion_r1165817522